db/error_handler.h

```diff
@@ -108,6 +109,9 @@ class ErrorHandler {
     // Used to store the context for recover, such as flush reason.
     DBRecoverContext recover_context_;
 
+    // The pointer of DB statistics.
+    std::shared_ptr<Statistics> bg_error_stats_;
+
```

Comment:
Should this be a shared or raw pointer?

Reply:
It is safer to use the shared pointer I think